pkg/core/function.go

```diff
@@ -43,28 +43,31 @@ import (
 )
 
 const (
-	functionLabel   = "riff.projectriff.io/function"
-	buildAnnotation = "riff.projectriff.io/nonce"
-	// annotation names with slashes are rejected :-/
+	functionLabel                 = "riff.projectriff.io/function"
+	buildAnnotation               = "riff.projectriff.io/nonce"
```

Comment:
I'm not suggesting we change this here, but I'd like to point out that this "revision count" isn't a (cryptographic) nonce. Also, "nonce" has an [unfortunate connotation](https://www.urbandictionary.com/define.php?term=nonce) in the UK.

Reply:
ouch

Reply:
Logged https://github.com/projectriff/riff/issues/1077.